rfcs/wpt-results-analysis.md

```diff
@@ -0,0 +1,19 @@
+# RFC 106: Move wpt.fyi metrics generation into WPT org
+
+## Summary
+
+Move https://github.com/Ecosystem-Infra/wpt-results and https://github.com/Ecosystem-Infra/wpt-results-analysis into the web-platform-tests GitHub org.
+
+## Details
+
+The following metrics on wpt.fyi are generated by wpt-results-analysis:
+
+- Browser Specific Failures
+- Compat 2021
+- Interop 2022
+
+wpt-results-analysis depends on wpt-results, a highly compressed form of all full wpt.fyi runs ever, currently ~52k stored in a ~385M Git repository. This makes it possible to load thousands (or all) runs into memory at the same time, speeding up metrics generation.
```

Comment:
Can we rename `wpt-results`, as I think we don't want people thinking it's a canonical data-source? I'm much happier with making everyone treat wpt.fyi's API as the canonical source, and treat wpt-results as an internal implementation detail of result-analysis. Maybe `wpt-results-analysis-cache`?

Reply:
`wpt-results-analysis-cache` sounds good to me.

Ideally wpt.fyi itself would maintain a representation of results that can be efficiently queried, but that's would be a large undertaking.